network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/trial_and_error.html

```diff
@@ -0,0 +1,16 @@
+{% load i18n static %}
+<section class="tw-dark tw-flex tw-flex-col tw-justify-center tw-items-center tw-mb-7">
+  <img src="{% static "_images/youtube-regrets/regrets-reporter/our-research-desktop.png" %}" alt="">
+  <div class="container tw-max-w-3xl tw-mx-auto tw-mt-5 medium:tw-mt-7 tw-space-y-5">
+    {% blocktrans trimmed %}
+      <h2 class="tw-text-4xl medium:tw-text-[3.25rem] tw-leading-none">People take a trial-and-error approach to controlling their recommendations.</h2>
+      <p>In the qualitative portion of our study, we learned that people do not feel in control of their experience on YouTube, nor do they have clear information about how to curate their recommendations. Many people take a trial-and-error
+        approach to controlling their recommendations using YouTube’s hodgepodge of options, like “Dislike,” “Not Interested,” and other buttons. It rarely works. Said one user:</p>
+      <blockquote class="tw-text-white tw-font-zilla tw-text-[1.625rem] medium:tw-text-[1.75rem] tw-leading-8 tw-font-normal">“Nothing changed. Sometimes I would report things as misleading and spam and the next day it was back in. It
+        almost feels like the more negative feedback I provide to their suggestions the higher bulls**t mountain gets. <span class="tw-text-youtube-regrets-red">Even when you block certain sources they eventually return.</span>”
+      </blockquote>
+      <p class="tw-text-xl tw-font-bold tw-font-changa">When YouTube’s own controls fail, people take matters into their own hands, using tactics including VPNs, new devices, and altered behavior like not clicking on or hovering over
+        videos.</p>
+    {% endblocktrans %}
```

Comment:
**Question** I found the style from the other PRs with the translations inside the block level HTML elements easier to read and follow. Is there a specific reason the style switched in this file/PR?

Reply:
I've done it this way just to encompass more content for translation context. Wrapping more content in the one block also reduces the amount of times we have to do a `{% blocktrans %}` tag so I figured it might be easier to read / maintain haha. I'll leave it for now but if we really want to swap these then we can in a future PR. (Just so we can get this work in for QA)